src/libs/ReportActionsUtils.ts

```diff
@@ -635,6 +665,147 @@ function isNotifiableReportAction(reportAction: OnyxEntry<ReportAction>): boolea
     return actions.includes(reportAction.actionName);
 }
 
+function getMemberChangeMessageElements(reportAction: OnyxEntry<ReportAction>): MemberChangeMessageElement[] {
```

Comment:
I request a review of this pattern.

This function builds a list of "elements", which contain translated parts of a message with some markers, abstracted from how they are presented. This is a kind of a viewmodel.

We present them in two different ways:
- as an HTML fragment (for rendering)
- as a plain text (for copying to clipboard)

Reply:
[Vit approved here](https://expensify.slack.com/archives/C02NK2DQWUX/p1701174783842619?thread_ts=1701167737.341309&cid=C02NK2DQWUX)

Reply:
Jules approved the PR; we're good to go